modules/programs/micro.nix

```
@@ -31,13 +38,72 @@ in {
           for supported values.
         '';
       };
+
+      bindings = mkOption {
+        type = jsonFormat.type;
+        default = { };
+        example = literalExpression ''
+          {
+            "Alt-d" = "SpawnMultiCursor";
+            "Escape" = "RemoveAllMultiCursors";
+            "CtrlDown" = "None";
+            "CtrlUp" = "None";
+            "Shift-PageDown" = "SelectPageDown";
+            "Shift-PageUp" = "SelectPageUp";
+          }
+        '';
+        description = ''
+          Configuration written to
+          <filename>$XDG_CONFIG_HOME/micro/bindings.json</filename>. See
+          <link xlink:href="https://github.com/zyedidia/micro/blob/master/runtime/help/keybindings.md"/>
+          for supported values.
+        '';
+      };
+
+      ensurePlugins = mkOption {
+        type = types.listOf types.str;
+        default = [ ];
+        example = literalExpression ''
+          [
+            "aspell"
+          ]
+        '';
```

I think this example is simple enough that `literalExpression` is overkill:

```suggestion
        example = [ "aspell" ];
```